iron-component-page.html

```diff
@@ -643,8 +643,8 @@
 
         _analyzerChanged: function() {
           var analyzer = this._analyzer;
-          this._setDocElements(analyzer && analyzer.elements ? analyzer.elements : []);
-          this._setDocBehaviors(analyzer && analyzer.behaviors ? analyzer.behaviors : []);
+          this._setDocElements(analyzer ? analyzer.elementsForFolder(this.base) : []);
+          this._setDocBehaviors(analyzer ? analyzer.elementsForFolder(this.base) : []);
```

Comment:
1. Does elementsForFolder return both elements and behaviors? 
2. Is this an expensive operation, or does it just return a reference?